eventhubs/data-plane/azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/InMemoryLeaseManager.java

```diff
@@ -323,8 +326,7 @@ synchronized InMemoryLease getLease(String partitionId) {
 
         synchronized List<String> getPartitionIds() {
             ArrayList<String> ids = new ArrayList<String>();
-            this.inMemoryLeasesPrivate.keySet().forEach((key) ->
-            {
+            this.inMemoryLeasesPrivate.keySet().forEach((key) -> {
                 ids.add(key);
             });
```

Comment:
I would like to see this simplified to `this.inMemoryLeasesPrivate.keySet().forEach(ids::add);`